bin/inbox-api.py

```diff
@@ -68,18 +63,23 @@ def main(prod, start_syncback, enable_tracer, config, port, enable_profiler):
         config_path = os.path.abspath(config)
         load_overrides(config_path)
 
-    if prod:
-        start(port, start_syncback, enable_tracer, enable_profiler)
-    else:
-        preflight()
-        from werkzeug.serving import run_with_reloader
-
-        run_with_reloader(
-            lambda: start(port, start_syncback, enable_tracer, enable_profiler)
-        )
```

Comment:
The whole thing was broken because werkzeug removed `run_with_reloader` APIs at some point. It has now unified `werkzeug.serving.run_simple` API that accepts `use_reloader`.